packages/api/test/e2e/doughnut-v0.e2e.ts

```diff
@@ -21,6 +21,7 @@ import { Api } from '../../src/Api';
 import { Keypair } from '@plugnet/util-crypto/types';
 import { KeyringPair } from '@cennznet/util/types';
 import { Extrinsic } from '@cennznet/types/extrinsic';
+import { cryptoWaitReady } from '@cennznet/util/';
```

Comment:
dougnhut-maker requires different cryptoWaitReady(),  the one you import here is from `@plugnet/wasm-crypto`

Reply:
Hmmm. https://github.com/polkadot-js/common/blob/80db504a3b12df716373132d62fdfb1f3fe3f260/packages/util-crypto/src/index.ts#L32-L35 does this mean importing "@polkadot/util-crypto" does it automatically?

Reply:
@cennznet/util doesn't depends on `@polkadot/*`.